ChangeLog.d/sha256-armce-arm.txt

```diff
@@ -0,0 +1,7 @@
+Features
+    * Support Armv8 Crypto Extension acceleration for SHA-256
+      when compiling for Thumb or 32-bit Arm.
+New deprecations
+    * Rename the SHAxxx_USE_A64_CRYPTO_yyy config options to
+      SHAxxx_USE_ARMV8_CRYPTO_yyy. The old names may still be
```

Comment:
... and therefore if this should be `_USE_ARMV8A_`. (If that is changed, might be nicer to re-write the commit that made the change to `_USE_ARMV8_`, since no point having two renames in the project history)

Reply:
I've done as two commits as they are quite large changes